### PR TITLE
Add-design-authors-on-Individual-Design-pages-on-the-catalog

### LINKED ISSUE #1121: Add design authors on Individual Design pages on the catalog
### Current Behavior
When you go to any individual design page, we don't see who published the design even though we have that feature in the backend.

### Desired Behavior
Add the name of the publisher on the individual design page.

### Screenshots / Mockups

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and designs for Meshery Site in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR fixes #1121 
- ## Adds the name of the publisher on the individual design page:
Screenshot 2023-05-09 012836
<br>
Screenshot 2023-05-09 013121
<br>
Screenshot 2023-05-09 003617

 <br>

- ## Fixes error in the yaml syntax in the file : [collections/_catalog/deployment/2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.md](https://github.com/meshery/meshery.io/blob/master/collections/_catalog/deployment/2df29b2b-7e5e-4d8b-a5b3-3dea569f4d25.md)
Screenshot 2023-05-09 012807

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Nice!

File: _includes/type.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
 		</span>
     </div>
 	{% endfor %}
+	<strong style="color: var(--color-secondary-dark);">Created by: </strong><em><span style="color: var(--color-secondary-dark);">{{include.post.userName}}</span></em>
+	<br>
```
[the remark this answers is not in the document]
Comment (vishalvivekm) [reply _includes/type.html:14]: in [.github/workflows/add-catalog.yml](https://github.com/meshery/meshery.io/blob/master/.github/workflows/add-catalog.yml): userId
```
            userId=$(jq -r .[$idx].user_id temp.json)
            userInfo=$(curl -sS "$MESHERY_CLOUD_BASE_URL/api/identity/users/profile/$userId" 2>./err1.txt)
            if [[ "$(<err1.txt)" != "" ]]; then
            >&2 echo "Error fetching User details"
            >&2 echo "$(<err1.txt)"
            fi
            echo "$userInfo" > info.json
```
Tried to do ```<a href="https://cloud.meshery.io/user/{{include.post.userId}}">{{include.post.userName}}</a>``` :
Screenshot 2023-05-09 022830
but the hyperlink on Abhishek Kumar resolves to https://cloud.meshery.io/user/ and takes to dashboard. FYI

[the remark this answers is not in the document]
Comment (leecalcote) [reply _includes/type.html:14]: @asubedy can help ensure the user id is available.

[the remark this answers is not in the document]
Comment (leecalcote) [reply _includes/type.html:14]: @vishalvivekm the user id is now available...

[the remark this answers is not in the document]
Comment (vishalvivekm) [reply _includes/type.html]: Thanks @leecalcote for the commit, just the D needs to be in lowercase ( ~~UserID~~ : userId)

**Comment by leecalcote** [comment]
👏

**Pull request merged**